tox.ini

```diff
@@ -40,9 +40,9 @@ commands=
     native-state-constantinople: py.test {posargs:tests/json-fixtures/test_state.py -k Constantinople}
     native-state-metropolis: py.test {posargs:tests/json-fixtures/test_state.py -k Metropolis}
 deps = -r{toxinidir}/requirements-dev.txt
-    coincurve
-    database: leveldb
-    trinity: leveldb
+    coincurve>=7.0.0,<8.0.0
+    database: leveldb>=0.194,<1.0.0
+    trinity: leveldb>=0.194,<1.0.0
```

Comment:
Or move these to the extras config, so we don't have to duplicate the version numbers from setup.py.

```
extras = 
    coincurve
    database: leveldb
    trinity: trinity
```